collection_requirements.rst

```diff
@@ -189,6 +189,20 @@ Also:
 
 See `Migrating content to a different collection <https://docs.ansible.com/ansible/devel/dev_guide/developing_collections.html#migrating-ansible-content-to-a-different-collection>`_ for complete details.
 
+Deprecation, redirection, and removing of moved content
```

Comment:
Should this go here, or in https://docs.ansible.com/ansible/devel/dev_guide/developing_collections.html#migrating-ansible-content-to-a-different-collection ?

Reply:
good point. I think the whole paragraph there should be adjusted. Will do and put the reference here.

Reply:
moved to https://github.com/ansible/ansible/pull/71854
the link is already there, so i'll close the PR